File: 6KYU/Take a Ten Minutes Walk/solution.py
def is_valid_walk(walk):
    nesw = {
        'n': 0,
        'e': 0,
        's': 0,
        'w': 0,
    }
    
    for d in walk:
        if d in nesw:
            nesw[d] = nesw[d]+1
    
    return len(walk) == 10 and nesw['n'] == nesw['s'] and nesw['w'] == nesw['e']

File: 6KYU/Take a Ten Minutes Walk/solution.jl
function isvalidwalk(walk)
    nesw = Dict('n' => 0, 'e' => 0, 's' => 0, 'w' => 0)
    
    for i in walk
        if haskey(nesw, i)
          nesw[i] = nesw[i]+1
        end
    end
    return length(walk) == 10 && nesw['n'] == nesw['s'] && nesw['e'] == nesw['w'];
end